concepticondata/conceptlists/Ruhlen-1998-36.tsv

```diff
@@ -0,0 +1,37 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Ruhlen-1998-36-1	1	birch bark		
+Ruhlen-1998-36-2	2	boat	1844	BOAT
+Ruhlen-1998-36-3	3	boil (v.)	2493	BOIL
+Ruhlen-1998-36-4	4	bow (n.)	994	BOW
+Ruhlen-1998-36-5	5	breast	1402	BREAST
+Ruhlen-1998-36-6	6	cedar		
+Ruhlen-1998-36-7	7	cheek	1730	CHEEK
+Ruhlen-1998-36-8	8	children	2099	CHILD
+Ruhlen-1998-36-9	9	clay	1399	CLAY
+Ruhlen-1998-36-10	10	deer	1936	DEER
+Ruhlen-1998-36-11	11	dry	1398	DRY
+Ruhlen-1998-36-12	12	elbow	981	ELBOW
+Ruhlen-1998-36-13	13	fir	1915	FIR
+Ruhlen-1998-36-14	14	foot	1301	FOOT
+Ruhlen-1998-36-15	15	guts	1334	GUTS
+Ruhlen-1998-36-16	16	he	1211	HE
+Ruhlen-1998-36-17	17	head	1256	HEAD
+Ruhlen-1998-36-18	18	hunger	1864	HUNGER
+Ruhlen-1998-36-19	19	lake	624	LAKE
+Ruhlen-1998-36-20	20	name	1405	NAME
+Ruhlen-1998-36-21	21	night (spend the -)	1233	NIGHT
+Ruhlen-1998-36-22	22	nit	267	NIT
+Ruhlen-1998-36-23	23	old	1229	OLD
+Ruhlen-1998-36-24	24	owl	735	OWL
+Ruhlen-1998-36-25	25	rabbit	1136	RABBIT
+Ruhlen-1998-36-26	26	river	666	RIVER
+Ruhlen-1998-36-27	27	rope	1218	ROPE
+Ruhlen-1998-36-28	28	shoulder	1482	SHOULDER
+Ruhlen-1998-36-29	29	(animal) skin	2614	SKIN (ANIMAL)
+Ruhlen-1998-36-30	30	snow (on ground)		
+Ruhlen-1998-36-31	31	snow (falling)		
+Ruhlen-1998-36-32	32	squirrel	1628	SQUIRREL
+Ruhlen-1998-36-33	33	stone	857	STONE
+Ruhlen-1998-36-34	34	summer	1342	SUMMER
+Ruhlen-1998-36-35	35	utensil		
```

Comment:
TOOL, perhaps?

Reply:
yes, why not?

Reply:
The gloss occurs in two lists (by the same [author](https://concepticon.clld.org/values/Mitterhofer-2013-300-106)) and it is mapped to 738 THING.

Reply:
To me, the meaning of utensil seems a bit broader than that of tool, but I'm not a native speaker. TOOL seems better fitting than THING, though.

Reply:
I'm glad a discussion came up here because I was wondering about the same. THING seems too broad to me as well, but if you agree on TOOL, I'll map it to that.